src/main/java/org/maveric/quarkus/panache/services/SavingAccountServices.java

```diff
@@ -113,7 +133,92 @@ else if (isNumeric(search)) {
         return responseDto;
     }
 
+    public SavingAccountResponseDto createAccount(FileUpload file, SavingAccountRequestDto savingAccountRequestDto) throws SQLException, IOException, CustomerProxyException {
+        System.out.println("****inside create account mock account="+new SavingAccount());
+
+        SavingAccount savingAccount = createAccountObject(file, savingAccountRequestDto);
+
+        SavingAccountResponseDto responseDto = null;
```

Comment:
Rename all SavingAccount to SavingsAccount

Reply:
Renamed all files